packages/core/src/app/payment/Payment.tsx

```diff
@@ -352,7 +352,8 @@ class Payment extends Component<
             selectedMethod.gateway === PaymentMethodId.Clearpay ||
             selectedMethod.gateway === PaymentMethodId.Checkoutcom ||
             selectedMethod.gateway === PaymentMethodId.Mollie ||
-            selectedMethod.gateway === PaymentMethodId.StripeV3
+            selectedMethod.gateway === PaymentMethodId.StripeV3 ||
```

Comment:
There is a TODO here https://github.com/bigcommerce/checkout-js/pull/2221/files#diff-01ca856011dd0cc1233e0450e9ece93e798fe8d07af017fa78240e27559f5dd7R324 which seems like will never get resolved. Is there a plan to address this at some point?